### ISSUE TITLE
Add event

### ISSUE THREAD

**Issue by rdealmeida12** [dszymczuk/dsEventCalendar57#3]
concrete 5.8.1 Installed calendar 3.4
 when adding event does not display calendar Date picker ( see attached )
It does allow you to put in a date but when the box loses focus it changes to the current date..
Same with the time input box ( see attached )
any ideas ??
Thanks
Bob

<img width="101" alt="timepicker" src="https://cloud.githubusercontent.com/assets/8337231/25565219/4a7bbbcc-2d91-11e7-8f8f-ac052c70ce91.PNG">
<img width="199" alt="datepicker" src="https://cloud.githubusercontent.com/assets/8337231/25565220/4a7f1a42-2d91-11e7-9299-d82980ab35cc.PNG">

**Comment by tigerxy** [comment]
Have you tried a different browser or deaktivate addons like adblock? Same result there? Any errors on the console of your browser?

**Comment by rdealmeida12** [comment]
didn't even think of adblocker.. whitelisted on domain but no luck
in the console found 11 errors all the same..
Uncaught TypeError: Cannot read property 'dayOfWeek' of undefined
    at ddfac585feb556f3a34183068b76d7bfe0968e35.js:3
Thanks for the help

Bob

**Comment by rdealmeida12** [comment]
Also tried it on IE and Firefox with no joy.. same problem

Bob

**Comment by dszymczuk** [comment]
Could you disable all addons in Concrete 5 and try again ?
Also change (if it is possible) to default theme and check again.

**Comment by rdealmeida12** [comment]
No addons running and I'm using the cloneamental theme which is elemental

Bob

**Comment by rdealmeida12** [comment]
I removed , downloaded and reinstalled the block. I noticed that after the block was installed on a page that the " Publish Changes" button changed.. ( view snippet..) 
<img width="206" alt="capture" src="https://cloud.githubusercontent.com/assets/8337231/25727983/3fcf1504-30fa-11e7-8965-3d23cfe2b1fd.PNG">

Bob

**Comment by tigerxy** [comment]
Hey Bob,

It appears that the javascript of the drop-down crashes because jquery.datetimepicker does not support your language. Can you type the following line into the console from the browser and post the response?
`Navigator.language || Navigator.userLanguage; `

You can also try the examples on this page. These should also not run.
[Datetimepicker page](http://xdsoft.net/jqplugins/datetimepicker/)

The current version of the plugin is v2.5.4. This addon uses version v2.4.1. If the examples work on the site, you can update the javascript file manually.

**Comment by rdealmeida12** [comment]
The examples worked perfectly.  I used the defaults..english
<img width="234" alt="timepicker" src="https://cloud.githubusercontent.com/assets/8337231/25776902/b5b02c4a-3299-11e7-94c9-87ce1114482c.PNG">

**Comment by tigerxy** [comment]
Try to replace this file: `packages / dsEventCalendar / js / jquery.datetimepicker.min.js`
with the newest you can find here: https://github.com/xdan/datetimepicker/releases/tag/2.5.4
Then clear the cache of concrete5.

**Comment by rdealmeida12** [comment]
Nope.. no luck.. Cleared cache.. uninstalled and reinstalled... tried on chrome (v58), Firefox, IE and Edge...
even tried on Ubuntu linux.. 
Changed language ( no english ... just english  GB ) no luck.. All show the same error..
Uncaught TypeError: i.dateFormat is not a function

**Comment by dszymczuk** [comment]
Is it possible to show this page ?
You can send on mail:  I will try to help you.

**Comment by rdealmeida12** [comment]
Hi Damian
Sorry for the delay in getting back to you, but things have been crazy
around here..
Anyway the first image is when I try to add an event, second is trying to
add time
The third image is when the block is installed on the frontend and the last
is without the block installed.
I'm using concrete 5.8.1 and there are no other blocks installed

Bob

**Comment by tom-rice** [comment]
This seems to be a problem with changes to variable handling in PHP 7 - the variable variables are no longer working so the $lang_datepicker_dsECS variable is no longer set and lang is empty, causing the error.

This fixed it for me:
controllers/single_page/dashboard/event_calendar/event.php line 108:
CHANGE:
$$s['opt'] = $s['value'];
TO:
${$s['opt']} = $s['value'];
